features/step_definitions/sriov.rb

```diff
@@ -0,0 +1,122 @@
+Given /^the sriov operator is running well$/ do
+  ensure_destructive_tagged
+  ensure_admin_tagged
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/evaluation of `cluster_version('version').channel.split('-')[1]` is stored in the :ocp_cluster_version clipboard/
+
+  unless project('openshift-sriov-network-operator').exists?
+    if cb.ocp_cluster_version.include? "4.[34]."
+      # create 4.3/4.4 namespaces
+      namespace_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/namespace/43/ns.yaml"
+      @result = admin.cli_exec(:create, f: namespace_yaml)
+      raise "Error creating namespace for openshift-sriov-network-operator" unless @result[:success]
+    else
+      # Create namespace for 4.5 and later
+      namespace45_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/namespace/45/ns45.yaml"
+      @result = admin.cli_exec(:create, f: namespace45_yaml)
+      raise "Error creating namespace for openshift-sriov-network-operator" unless @result[:success]
+    end
+    
+  end
+  step %Q/I use the "openshift-sriov-network-operator" project/
+  unless operator_group('sriov-network-operators').exists?
+    og_yaml ||= "#{ENV['BUSHSLICER_HOME']}/testdata/networking/sriov/og/og.yaml"
```

Comment:
@zhaozhanqi please use the I obtain the datafile step

Reply:
@pruan-rht updated. thanks